consensus.go

```diff
@@ -899,3 +944,22 @@ func MaxFaultyNodes(nodesCount int) int {
 func QuorumSize(nodesCount int) int {
 	return 2*MaxFaultyNodes(nodesCount) + 1
 }
+
+func totalVotingPower(mp map[NodeID]uint64) uint64 {
```

Comment:
Maybe this can be TotalVotingPower so we can use it in V3 code ?? Just a suggestion.

Reply:
Aggree. Done